### DISCUSSION TITLE
水

### DISCUSSION THREAD

**Discussion by xingyun-kb** [share121/inter-knot#4513] [灌水]
![Uploading 1724245087878.gif…]()

**Comment by cpu644** [comment]
你图没上传完

**Comment by xingyun-kb** [comment]
![Uploading 1724244420563.gif…]()

**Comment by xingyun-kb** [comment]
哈哈哈<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/829f0a1d533765b80769dd1a38dc338c_9182108590963360703.png" alt="阿姬惊讶" width="75" >
